src/wp-includes/class-wp-block-bindings-registry.php

```diff
@@ -159,7 +220,7 @@ public function unregister( string $source_name ) {
 		if ( ! $this->is_registered( $source_name ) ) {
 			_doing_it_wrong(
 				__METHOD__,
-				/* translators: %s: Block bindings source name. */
+				// translators: %s: Block bindings source name.
```

Comment:
Can we revert the changes applied to the hints provided to translators? It looks like they all use the `/* */` syntax for some reasons I don't fully understand.

Reply:
Just did it in [this commit](https://github.com/WordPress/wordpress-develop/pull/6079/commits/59fde2951385ae6af133e44dc334be65a6745ccb).